### PR TITLE
Replaced number type of id with string one using graphql-codegen

### PR DESCRIPTION
**Pull request by YoshinariYamanaka**
# Description
*

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -23,12 +23,12 @@ export default function Top({ searchParams }: { searchParams: { q: string } }) {
...
 
   const searchWord = searchParams.q
-  const uuid = useUserUuid()
+  const userId = useUserId()
 
   const { data, loading, fetchMore, refetch } = useTripsCollectionQuery({
     variables: {
       filter: {
-        uuid: { eq: uuid },
+        id: { eq: userId },
```
Comment (samuraikun) [inline app/page.tsx:31]: We need to use `user_id` parameter to fetch trips.
Could you fix to use `user_id`?

```suggestion
        user_id: { eq: userId },
```

Comment (samuraikun) [inline app/page.tsx]: We need to use `user_id` parameter to fetch trips.
Could you fix to use `user_id`?

```suggestion
        user_id: { eq: userId },
```

**Review by samuraikun — CHANGES_REQUESTED** [review]

File: app/page.tsx (hunk 1 above)
Comment (YoshinariYamanaka) [reply app/page.tsx]: ohh yes!! it's trip id... 
Thx !

Comment (YoshinariYamanaka) [reply app/page.tsx]: Fixed !

**Review by samuraikun — APPROVED** [review]
Thank you 🍎 
[LGTMeow](https://lgtmeow.com)

File: app/trip/[id]/components/trip-details-header.tsx [hunk 2]
```diff
@@ -21,11 +21,11 @@ type TripDetailsHeaderProps = {
   dateFrom: string | null | undefined
   dateTo: string | null | undefined
   users: {
-    id: number | undefined
+    id: string | undefined
```
Comment (kanatagu) [inline app/trip/[id]/components/trip-details-header.tsx:24]: Thank you 🙏

**Review by kanatagu — APPROVED** [review]
LGTM 🎉

**Pull request merged**